Keas.Mvc/Services/RolesSessionsManager.cs

```diff
@@ -0,0 +1,170 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Keas.Core.Data;
+using Microsoft.AspNetCore.Http;
+using Microsoft.EntityFrameworkCore;
+using Newtonsoft.Json;
+using Serilog;
+
+namespace Keas.Mvc.Services
+{
+    public interface IRolesSessionsManager
+    {
+        Task<string[]> GetTeamRoleNames(string slug);
+        Task<string[]> GetTeamOrAdminRoleNames(string slug);
+        Task<string[]> GetSystemRoleNames();
+        void ClearSessionRoles();
+
+    }
+
+    public class RolesSessionsManager : IRolesSessionsManager

```

Comment:
For the implementation, i don't like the double plural.  There aren't multiple sessions so maybe just `RolesSessionManager`?

Reply:
My concern around RolesManager is that name already exists for the EF Roles Manager.